waypoint_placement:
    #strategy: "ergodic"
    #strategy: "random"
    strategy: "greedy"
graph_planner:
    #strategy: "ergodic"
    #strategy: "random"
    strategy: "greedy"
